NanoMQ compile failed on mac

NanoMQ compile failed on mac, here is the cmd and log
```
$ cmake -G Ninja -DCFG_METHOD=CMAKE_CONFIG -DCMAKE_BUILD_TYPE=Debug -DNOLOG=ON -DMQ=OFF  ..
$ ninja


[1/2] Building C object nanomq/CMakeFiles/nanomq.dir/rest_api.c.o
FAILED: nanomq/CMakeFiles/nanomq.dir/rest_api.c.o
/usr/bin/clang -DNNG_STATIC_LIB -DNOLOG -DPARALLEL=32 -DSUPP_CLIENT -I../nanomq/. -I../nanomq/include -I../nng/include/nng -I../nanolib/include -I../nng/include -std=gnu99 -g -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX10.15.sdk -MD -MT nanomq/CMakeFiles/nanomq.dir/rest_api.c.o -MF nanomq/CMakeFiles/nanomq.dir/rest_api.c.o.d -o nanomq/CMakeFiles/nanomq.dir/rest_api.c.o -c ../nanomq/rest_api.c
../nanomq/rest_api.c:678:9: error: implicit declaration of function 'getcwd' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
                                if (getcwd(dir, sizeof(dir)) != NULL) {
                                    ^
../nanomq/rest_api.c:678:34: warning: comparison between pointer and integer ('int' and 'void *') [-Wpointer-integer-compare]
                                if (getcwd(dir, sizeof(dir)) != NULL) {
                                    ~~~~~~~~~~~~~~~~~~~~~~~~ ^  ~~~~
../nanomq/rest_api.c:697:9: error: implicit declaration of function 'getcwd' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
                                if (getcwd(dir, sizeof(dir)) != NULL) {
                                    ^
../nanomq/rest_api.c:697:34: warning: comparison between pointer and integer ('int' and 'void *') [-Wpointer-integer-compare]
                                if (getcwd(dir, sizeof(dir)) != NULL) {
                                    ~~~~~~~~~~~~~~~~~~~~~~~~ ^  ~~~~
../nanomq/rest_api.c:716:9: error: implicit declaration of function 'getcwd' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
                                if (getcwd(dir, sizeof(dir)) != NULL) {
                                    ^
../nanomq/rest_api.c:716:34: warning: comparison between pointer and integer ('int' and 'void *') [-Wpointer-integer-compare]
                                if (getcwd(dir, sizeof(dir)) != NULL) {
                                    ~~~~~~~~~~~~~~~~~~~~~~~~ ^  ~~~~
3 warnings and 3 errors generated.
ninja: build stopped: subcommand failed.

--- a/nanomq/rest_api.c
+++ b/nanomq/rest_api.c
@@ -20,6 +20,7 @@
 #include <stdint.h>
 #include <stdio.h>
 #include <string.h>
+#include <unistd.h>
 
 static http_msg error_response(
     http_msg *msg, uint16_t status, enum result_code code, uint64_t sequence);

